### ISSUE TITLE
jagger-operator 1.32.0 fails to start successfully

### ISSUE THREAD

**Issue by ayaranga** [jaegertracing/helm-charts#351]
**Describe the bug**
Trying to install jagger-operator 1.32.0 using jaeger-operator-2.30.0 helm chart.  Following error is seen in the jaeger-operator POD logs:
```
2022-04-11T01:33:17.406Z	ERROR	controller-runtime.manager.controller.jaeger	Could not wait for Cache to sync	{"reconciler group": "jaegertracing.io", "reconciler kind": "Jaeger", "error": "failed to wait for jaeger caches to sync: timed out waiting for cache to be synced"}
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.9.6/pkg/internal/controller/controller.go:195
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.9.6/pkg/internal/controller/controller.go:221
sigs.k8s.io/controller-runtime/pkg/manager.(*controllerManager).startRunnable.func1
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.9.6/pkg/manager/internal.go:696
2022-04-11T01:33:17.408Z	ERROR	controller-runtime.manager	error received after stop sequence was engaged	{"error": "failed to wait for deployment caches to sync: timed out waiting for cache to be synced"}
2022-04-11T01:33:17.408Z	ERROR	controller-runtime.manager	error received after stop sequence was engaged	{"error": "failed to wait for jaeger caches to sync: timed out waiting for cache to be synced"}
time="2022-04-11T01:33:17Z" level=fatal msg="open /tmp/k8s-webhook-server/serving-certs/tls.crt: no such file or directoryManager exited non-zero"
```

**To Reproduce**
1.  Install jaeger-operator using helm
     ```
      git checkout jaeger-operator-2.30.0
      cd charts/jaeger-operator
      helm install jaeger-operator .  --set rbac.clusterRole=true
      ```
2. Check the jaeger-operator logs
```
  export POD=$(kubectl get pods -l app.kubernetes.io/instance=jaeger-operator -lapp.kubernetes.io/name=jaeger-operator --namespace default --output name)
  kubectl logs $POD --namespace=default
```

**Expected behavior**
Expected to see no errors in the jaeger-operator logs.

**Screenshots**
NA

**Version (please complete the following information):**
 - OS: [e.g. Linux] Linux
 - Jaeger version: [e.g. 1.8] NA
 - Deployment: [e.g. bare metal, Docker, Kubernetes, ...]  Kubernetes (kind version v0.11.1 go1.16.4 linux/amd64)

**What troubleshooting steps did you try?**
Not found any relevant troubleshooting steps in https://www.jaegertracing.io/docs/latest/troubleshooting/ for the error seen above.
Tried to install using jaeger-operator.yaml file specified at https://www.jaegertracing.io/docs/1.32/operator/#installing-the-operator-on-kubernetes. The operator comes up successfully. Issue is seen only when using the helm chart.

**Additional context**
The same steps work for installing jaeger-operator 1.30.0 using jaeger-operator-2.29.0 helm chart.

**Comment by ayaranga** [comment]
@LP0101, checking with you as you updated the helm chart for 1.32.0. Did you face the above issue? If so, can you please share the steps for resolution.

**Comment by LP0101** [comment]
It looks like I unknowingly tested the upgrade in a cluster with the latest CRDs already installed, so I didn't catch that they were updated :(

If it's not done by then, I'll open a new PR with the fixes ASAP tomorrow morning.

**Comment by ayaranga** [comment]
Thanks @LP0101 . Hope the templates also would be updated to have certmanager and webhook changes required for 1.32.0

**Comment by ayaranga** [comment]
@LP0101 Any update on this?

**Comment by AlKapkone** [comment]
Hello, I've got the same issue. Any updates on this?

**Comment by stevehipwell** [comment]
The CRD in the chart is very old and the chart hasn't been updated to use cert-manager, as is required by the new CRD. I don't get who approved this release?

**Comment by NicolasKarnick** [comment]
Any news on this @LP0101 ?

**Comment by rafilkmp3** [comment]
+1

**Comment by AlKapkone** [comment]
Any updates?

**Comment by bhargavmg** [comment]
Still facing this issue. Any updates or anyone managed to resolve this ?

**Comment by Madaspe** [comment]
any update? I've got the same issue

**Comment by dimitarshenkov** [comment]
same

**Comment by JohnFrampton** [comment]
Same for me.

**Comment by scDisorder** [comment]
Same

**Comment by avishefi** [comment]
Same for me. Is there an expected update schedule? Looks like the kustomize manifests are up to date due to Kubebuilder but the Helm chart for jaeger-operator fell behind.

**Comment by djsly** [comment]
any maintainer for this chart ? have we reached a point that this chart just reached its end of life ?

**Comment by Oceanswave** [comment]
Any update? Is Jaeger dead?

**Comment by avishefi** [comment]
@Oceanswave work is under way:
* https://github.com/jaegertracing/helm-charts/pull/367
* https://github.com/jaegertracing/helm-charts/pull/368

**Comment by czomo** [comment]
This should be resolved now. Check  `2.32.2` version of the helm chart. Before that look at prerequisites. Additional compability matrix [here](https://github.com/jaegertracing/helm-charts/blob/main/charts/jaeger-operator/COMPATIBILITY.md)

**Comment by Oceanswave** [comment]
@czomo Thank you!

**Comment by ayaranga** [comment]
Thanks @czomo. It is working fine now. I have a question regarding the compatibility. Can we use the 2.32.2 helm chart for installing jaeger-operator 1.32.0 as well? I tried the following and it seemed to work fine. Looking for a confirmation for the same.
```
helm install jaeger-operator jaegertracing/jaeger-operator --set rbac.clusterRole=true --set image.tag=1.32.0
```

**Comment by czomo** [comment]
@ayaranga  Despite fact it may work i don’t  recommend it. It is untestested and you would need to match CRD to corresponding version of operator. If you want to do it or your own please adapt it to [Crd](https://github.com/jaegertracing/jaeger-operator/blob/a1fa30b070e0c59d5ee5cdf8d3b9750488440176/config/crd/bases/jaegertracing.io_jaegers.yaml#L2)  whatever version you want.

**Comment by ayaranga** [comment]
@czomo Got it, thanks. So, there is no helm-chart version currently available that supports jaeger-operator 1.32.0. Are there any plans to patch 2.31.0 to make it work with 1.32.0?

**Comment by czomo** [comment]
> @czomo Got it, thanks. So, there is no helm-chart version currently available that supports jaeger-operator 1.32.0. Are there any plans to patch 2.31.0 to make it work with 1.32.0?

This is question for maintainers tho, in my opinion there is no sense to patch it when [1.35](https://github.com/jaegertracing/jaeger-operator/blob/main/CHANGELOG.md#1350-2022-06-16) is out
